ui/package.json

```diff
@@ -97,7 +98,7 @@
     "ember-inline-svg": "^1.0.1",
     "ember-load-initializers": "^2.1.2",
     "ember-maybe-import-regenerator": "^1.0.0",
-    "ember-modifier": "^3.1.0",
+    "ember-modifier": "3.2.7",
```

Comment:
`question`:  `3.2` was a major change to this [library](https://github.com/ember-modifier/ember-modifier/releases/tag/v3.2.0). Did you check the changelog and ensure we won't have any breaking changes (either from us or dependencies)?

Reply:
`blocking`: I think we need to drop support for `node: 12` as a result.

I get the following error message on build:

`The engine "node" is incompatible with this module. Expected version ">=14". Got "12.22.1"`.

We should update the `node` version in the `engines` stanza here to alert our users.

cc:  @DingoEatingFuzz

Reply:
Tested against our resize, code-mirror, and autofocus existing modifiers. I think we're good here; we'll need to do some migrations before 4.0.